src/Box.hs

```diff
@@ -247,7 +247,7 @@ initRow startColumn =
 
 spacesInTab :: Int
 spacesInTab =
-  4
+  2
```

Comment:
I think number of spaces per tab should be configured. 4 is ok as a default value, but it would be fair to let others to choose...